### PR TITLE
[WIP] init theming updates

### PR DESCRIPTION
**Pull request by marcellamaki**
#### Issue addressed

Addresses #*PR# HERE*

## Steps to test

1. Step 1
2. Step 2
3. ...

## Testing checklist

- [ ] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Critical and brittle code paths are covered by unit tests
- [ ] The change has been added to the `changelog`

## Reviewer guidance

- [ ] Is the code clean and well-commented?
- [ ] Are there tests for this change?
- [ ] Are all UI components LTR and RTL compliant (if applicable)?
- [ ] _Add other things to check for here_

### CODE REVIEW

**Comment by MisRob** [comment]
@marcellamaki Please rebase on top of the latest `develop` as soon as you continue this work

**Comment by MisRob** [comment]
@marcellamaki This will need to be rebased and retargeted to `main`

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,17 @@ Changelog is rather internal in nature. See release notes for the public overvie
 
 <!-- Put all new updates into this section, please -->
 
+- [#440]
+  - **Description:** Updates brand theme color tokens and token mapping within components
+  - **Products impact:** Brand token updates
+  - **Addresses:** - https://github.com/learningequality/kolibri-design-system/issues/545
+  - **Components:** -
+  - **Breaking:** -
```
Comment (MisRob) [inline CHANGELOG.md]: This will be a breaking update as it changes UI significantly

Comment (MisRob) [reply CHANGELOG.md]: I'd also appreciate if the changes could be listed here in a bit more detail, for example what categories of theming were affected. No need to explain which hex color changed into what, rather just to sum up the most important areas of impact.

Comment (LianaHarris360) [reply CHANGELOG.md]: Thank you for this, I was planning to update the changelog with more info but wasn't too sure about the level of detail to include.

Comment (MisRob) [reply CHANGELOG.md]: Thank you, new updates make sense. I see one place that could deserve more detail in my view

When you say
> new content-related token labels have been introduced and existing ones have been renamed.

Could you list the new token labels here, and also before/after labels of existing tokens? So we can find them in the products easily and rename without the need to dig in this PR. I think of this as changes in public API, basically (this is different from just tweaking an existing color - I don't think we need to spend a day on documenting all that).

**Comment by LianaHarris360** [comment]
Closing this PR and will open a new one with the same changes.

File: CHANGELOG.md (hunk 1 above)
Comment (MisRob) [reply CHANGELOG.md]: > but wasn't too sure about the level of detail to include.

Generally, you could just imagine that you're someone who's not familiar with the code diff in detail and tried to introduce new theming to Kolibri/Studio. Anything that would be useful for you to know will serve as good guidance :) as I will include it in the release notes. Thank you.

**Pull request closed without merging**